plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteJdbcPlugin.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.spi.Plugin;
+import io.trino.spi.connector.ConnectorFactory;
+import io.trino.testing.TestingConnectorContext;
+import org.testng.annotations.Test;
+
+import static com.google.common.collect.Iterables.getOnlyElement;
+
+public class TestIgniteJdbcPlugin
```

Comment:
Also add

- TestIgniteTypeMapping - see TestPostgreSqlTypeMapping for example to follow
- TestIgniteCaseInsensitiveMapping - see  TestPostgreSqlCaseInsensitiveMapping for example to follow
- TestIgniteClient -- see TestPostgreSqlClient for example to follow